lib/index.js

```diff
@@ -33,15 +33,13 @@ var CustomerIO = module.exports = integration('Customer.io')
  */
 
 CustomerIO.prototype.identify = function(identify, fn){
-  var self = this;
-  this.visit(identify, function(){
-    self
-    .put(identify.userId())
-    .auth(self.settings.siteId, self.settings.apiKey)
-    .type('json')
-    .send(mapper.identify(identify))
-    .end(self.handle(fn));
-  });
+  if (!identify.active()) return setImmediate(fn);
```

Comment:
Ah, I see. I didn't understand the semantics of `active()` correctly.

I've pushed up a commit that I think does what you suggested. Please let me know if there are any other changes needed!